src/central-intro.rst

```diff
@@ -17,28 +17,30 @@ Here are some of the major features we support today:
  - User accounts and management
```

Comment:
I was looking at this list and it seems weird that User accounts is first. I didn't think hard about this, but maybe replace the first three bullets with

```
Projects to organize users, permissions, and forms
- With flexible user accounts and management
- With support for role-based permissions
```

Reply:
well, users exist outside  projects too. but i'll move the ordering around.